data/raw/keybindings.json

```diff
@@ -3351,6 +3351,13 @@
     "name": "Remove keybinding",
     "bindings": [ { "input_method": "keyboard_any", "key": "-" }, { "input_method": "keyboard_code", "key": "KEYPAD_MINUS" } ]
   },
+  {
+    "type": "keybinding",
+    "id": "RESET",
+    "category": "HELP_KEYBINDINGS",
+    "name": "Reset keybinding",
+    "bindings": [ { "input_method": "keyboard_char", "key": "*" }, { "input_method": "keyboard_code", "key": "KEYPAD_MULTIPLY" } ]
```

Comment:
```suggestion
    "bindings": [ { "input_method": "keyboard_char", "key": "*" }, { "input_method": "keyboard_code", "key": "8", "mod": [ "shift" ] }, { "input_method": "keyboard_code", "key": "KEYPAD_MULTIPLY" } ]
```

Reply:
Sure, why not?
I am curious. What system will benefit from this? Where would it register the `*` correctly thanks to your addition?
On my Windows, with an EN keyboard, I perceive no change.

Reply:
This allows pressing `shift+8` for `*` on qwerty keyboard in the key code mode (which can be turned on in the settings). The key code mode can help prevent IMEs/dead keys from interfering with the input, but it is currently only supported for qwerty keyboard.